### PR TITLE
add facet configuration for cnp5

### PR DESCRIPTION
**Pull request by bitner**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -163,7 +164,7 @@ services:
             # For local dev'ing, uncomment the below
             #  If using the line below verbatim, 
             #  registry must be cloned to ~/boundless/registry
-            # - ~/boundless/registry:/opt/registry:ro
+            - ~/boundless/registry:/opt/registry:ro
```
Comment (arhote) [inline docker-compose.yml]: Did you mean to commit this one in?

File: docker-compose.yml [hunk 2]
```diff
@@ -172,7 +173,7 @@ services:
                 ipv4_address: 172.16.238.6
         environment:
             # Setup registry to use included search
-            - REGISTRY_SEARCH_URL=http://search:9200
+            # - REGISTRY_SEARCH_URL=http://search:9200
```
Comment (arhote) [inline docker-compose.yml]: Same here?

File: docker-compose.yml (hunk 1 above)
Comment (bitner) [reply docker-compose.yml]: nope

File: docker-compose.yml (hunk 2 above)
Comment (bitner) [reply docker-compose.yml]: nope

**Pull request merged**